Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM soroushojagh/lds:base
```

Comment:
For long term - is this ok for everyone to be pulling from your personal docker account? I'm wondering if it is worth typing the dockerfile out here that made this image so you don't need to keep this account alive forever.

Reply:
Yes it is ok and I will keep the base image in my docker hub for users. BTW, I will upload the base image somewhere in the installation folder and let users use it if anything happened in my docker hub.